assets/queries/azureResourceManager/email_notifications_set_off/query.rego

```diff
@@ -1,43 +1,46 @@
 package Cx
 
 CxPolicy[result] {
+	emailType := ["alertNotifications", "notificationsByRole"]
 	resource := input.document[i].resources[_]
 	resource.type == "Microsoft.Security/securityContacts"
-	object.get(resource.properties, "alertNotifications", "undefined") == "undefined"
+	object.get(resource.properties, emailType[x], "undefined") == "undefined"
 
 	result := {
 		"documentId": input.document[i].id,
 		"searchKey": sprintf("resources.name={{%s}}.type={{Microsoft.Security/securityContacts}}.properties", [resource.name]),
 		"issueType": "MissingAttribute",
-		"keyExpectedValue": "resource with type 'Microsoft.Security/securityContacts' has 'alertNotifications.state' property set to 'On'",
-		"keyActualValue": "resource with type 'Microsoft.Security/securityContacts' doesn't have 'alertNotifications' property defined",
+		"keyExpectedValue": sprintf("resource with type 'Microsoft.Security/securityContacts' has '%s.state' property set to 'On'", [emailType[x]]),
+		"keyActualValue": sprintf("resource with type 'Microsoft.Security/securityContacts' doesn't have '%s' property defined", [emailType[x]]),
 	}
 }
 
 CxPolicy[result] {
+	emailType := ["alertNotifications", "notificationsByRole"]
```

Comment:
maybe you could define `emailType := ["alertNotifications", "notificationsByRole"]` outside of the rule in order to prevent code repetition